### PR TITLE
feat : add kuala lumpur meetup

### PR DESCRIPTION
**Pull request by houssembaltii**
image url : https://unsplash.com/fr/photos/batiments-de-la-ville-pendant-la-nuit-hAPjYHczkMY

### CODE REVIEW

File: src/content/meetup/2024-09-26-kuala-lumpur/about.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                        className="font-semibold text-primary"
+                      >
+                        {formatDateTime(props.meetup.date)}.
+                      </time>{" "}
+                      Come enjoy conferences and a networking cocktail .
+                    </span>
+                  </li>
+                )}
+              </ul>
+              <p>Description for the location&apos;s meetup</p>
```
Comment (JeanneGrenet) [inline src/content/meetup/2024-09-26-kuala-lumpur/about.tsx:69]: We can't put in prod with this, no ? Put it in comment I think

File: src/content/meetup/2024-09-26-kuala-lumpur/about.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                      aria-hidden="true"
+                    />
+                    <span>
+                      <time
+                        dateTime={props.meetup.date}
+                        className="font-semibold text-primary"
+                      >
+                        {formatDateTime(props.meetup.date)}.
+                      </time>{" "}
+                      Come enjoy conferences and a networking cocktail .
```
Comment (JeanneGrenet) [inline src/content/meetup/2024-09-26-kuala-lumpur/about.tsx:64]: Typo :  Come enjoy conferences and a networking cocktail.

**Review by JeanneGrenet — CHANGES_REQUESTED** [review]

File: src/content/meetup/2024-09-26-kuala-lumpur.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: Fork it! Meetups Kuala Lumpur 2024
+name: Meetup Kuala Lumpur, Malaisia
+date: 2024-09-26
```
Comment (JeanneGrenet) [inline src/content/meetup/2024-09-26-kuala-lumpur.mdx:4]: Wrong date, you also have to change the names of the files

**Review by JeanneGrenet — CHANGES_REQUESTED** [review]

File: public/meetups/2024-10-22-kuala-lumpur.jpg
Comment (yoannfleurydev) [inline public/meetups/2024-10-22-kuala-lumpur.jpg:1]: The image is a bit heavy by default, you can reduce the size I think

File: src/app/meetups/[slug]/page.tsx [hunk 4]
```diff
@@ -60,14 +60,16 @@ export default async function MeetupPage({ params }: MeetupPageProps) {
...
+    offers: meetup.tickets?.offers
+      ? meetup.tickets?.offers.map((offer) => ({
+          "@type": "Offer",
+          price: offer.price,
+          priceCurrency: offer.priceCurrency,
+          url: meetup.tickets?.href,
+          availability: `https://schema.org/${offer.availability}`,
+          validFrom: offer.validFrom.toISOString(),
+        }))
+      : [],
```
Comment (yoannfleurydev) [inline src/app/meetups/[slug]/page.tsx:72]: please rebase on main to make sure you have the last version of the code I did in the PR for Hanoi

File: src/content/collections.ts [hunk 5]
```diff
@@ -173,15 +173,17 @@ const collections = {
...
+            .array(
+              z.object({
+                name: z.string(),
+                price: z.number().positive(),
+                priceCurrency: z.string(),
+                availability: z.enum(["InStock", "SoldOut", "PreOrder"]),
+                validFrom: z.date(),
+              }),
+            )
+            .optional(),
```
Comment (yoannfleurydev) [inline src/content/collections.ts:186]: same comment as before, rebase on `main` to make sure you have the latest code

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]

**Pull request merged**